data/sql/updates/pending_db_world/rev_1588196804255396600.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1588196804255396600');
+
+DROP TABLE IF EXISTS `map_difficulty_data`;
+CREATE TABLE IF NOT EXISTS `map_difficulty_data` (
+  `MapId` INT(10) UNSIGNED NOT NULL DEFAULT 0,
+  `Difficulty` INT(10) UNSIGNED NOT NULL DEFAULT 0,
+  `ParentDifficulty` INT(10) UNSIGNED NOT NULL DEFAULT 0,
+  `Comment` TEXT,
+  PRIMARY KEY (`MapId`, `Difficulty`)
+) ENGINE=MYISAM;
+
```

Comment:
is this table only used to set epic difficulty? if so then the name should reflect that else it's confusing

Reply:
also the columns naming convention should follow the table naming convention aka lowercase with _

Reply:
No, it does exactly what it says ;-)
It extends the map_difficulty_data by the entries

Reply:
but there is only one difficulty

it is just straight confusing

people might think it's related to heroic or raid size dunno